homeassistant/components/tasmota_irhvac/manifest.json

```
@@ -0,0 +1,11 @@
+{
+    "domain": "tasmota_irhvac",
+    "name": "Tasmota Irhvac",
+    "documentation": "https://github.com/hristo-atanasov/Tasmota-IRHVAC",
```

The documentation needs to be hosted on [home-assistant.io](https://www.home-assistant.io) therefore you will need to open a PR [here](https://github.com/home-assistant/home-assistant.io) to add this documentation :)
```suggestion
    "documentation": "https://www.home-assistant.io/integrations/tasmota_irhvac",
```

---

Thanks for your help, man .. This is the first time I'making PR here and I'm not so much in the situation .. :)

---

No worries 😄 you should take a look at the [development checklist](https://developers.home-assistant.io/docs/en/development_checklist.html) which lists requirements for new integrations and other contributions toi help you solve common problems.

---

I have one question .. I tried to add myself as code owner, but my username contains "-" in the middle and Visual Studio Code shows it as error .. Is it really error or I can add myself? Or just leave it blank? I don't need to be "famous" .. :)

---

>  Is it really error or I can add myself?

No, that should not be an error, so please add yourself :) Visual Studio Code should only complain if the codeowner name is not enclosed in "".

---

@springstan  I'm running hassio (Home Assistant) on VirtualBox .. I'm not sure how to run
"python3 -m script.gen_requirements_all", "codespell" and "black --fast homeassistant tests" ..

---

I made everything, but I still have these 2 checks that were not successful. What to do now?